app/src/main/java/com/hammer/talkbbokki/presentation/comment/CommentsScreen.kt

```diff
@@ -0,0 +1,265 @@
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.text.style.TextAlign
+import androidx.compose.ui.unit.dp
+import androidx.hilt.navigation.compose.hiltViewModel
+import com.hammer.talkbbokki.R
+import com.hammer.talkbbokki.ui.theme.*
+
+@Composable
+fun CommentsRoute(
+    onBackClick: () -> Unit,
+    onRecommentClick: () -> Unit,
+    viewModel: CommentsViewModel = hiltViewModel()
+) {
+    val comments = viewModel.commentItems.collectAsState()
+    CommentsScreen(
+        onBackClick,
+        onRecommentClick,
+        { viewModel.postComment(it) },
+        comments.value
+    )
+}
+
+@Composable
+fun CommentsScreen(
+    onBackClick: () -> Unit,
+    onRecommentClick: () -> Unit,
+    onClickPostComment: (String) -> Unit,
+    comments: List<Comment>
+) {
+    Box(
+        Modifier
+            .fillMaxSize()
+            .background(MainBackgroundColor)
+    ) {
+        Column(
+            Modifier
+                .fillMaxSize()
+                .padding(bottom = 78.dp)
+        ) {
+            CommentsHeader(onBackClick)
+            if (comments.isEmpty()) {
+                CommentEmpty()
+            } else {
+                CommentList(
+                    comments,
+                    onRecommentClick
+                )
+            }
+        }
+        Box(Modifier.align(Alignment.BottomCenter)) {
+            CommentInputArea { onClickPostComment(it) }
+        }
+    }
+}
+
+@Composable
+fun CommentsHeader(
+    onBackClick: () -> Unit,
+) {
+    val replyCount = 0
+    Box(
+        Modifier
+            .height(56.dp)
+            .fillMaxWidth()
+    ) {
+        Icon(painter = painterResource(id = R.drawable.ic_arrow_left),
+            tint = White,
+            contentDescription = null,
+            modifier = Modifier
+                .clickable {
+                    onBackClick()
+                }
+                .align(Alignment.CenterStart)
+                .padding(start = 20.dp))
+        Text(
+            text = "댓글 ($replyCount)", Modifier.align(Alignment.Center), color = White
+        )
+    }
+}
+
+@Composable
+fun CommentItem(
+    nickname: String,
+    date: String,
+    content: String,
+    replyCount: Int,
+    onDeleteClick: () -> Unit,
+    onRecommentClick: () -> Unit
+) {
+    val isMine = false
+    Column(modifier = Modifier.padding(20.dp)) {
+        Row(verticalAlignment = Alignment.CenterVertically) {
+            Text(
+                text = nickname, style = TalkbbokkiTypography.b3_bold,
+                color = if (isMine) MainColor01 else White
+            )
+            Spacer(modifier = Modifier.width(8.dp))
+            Text(text = date, style = TalkbbokkiTypography.caption, color = Gray06)
+        }
+        if (isMine) {
+            Icon(painter = painterResource(id = R.drawable.ic_close),
```

Comment:
🚫 **[ktlint]** <standard:argument-list-wrapping> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Argument should be on a separate line (unless all arguments can fit a single line)